src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/TreeView/GroupedByTargetTreeViewProviderTests.cs

```diff
@@ -703,18 +703,18 @@ public class GroupedByTargetTreeViewProviderTests
             // Assert            
             var expectedFlatHierarchy =
 @"Caption=MyDependencies, FilePath=, IconHash=325248080, ExpandedIconHash=325248080, Rule=, IsProjectItem=False, CustomTag=
+Caption=ZzzDependencyRoot, FilePath=ZzzDependencyRoot, IconHash=0, ExpandedIconHash=0, Rule=, IsProjectItem=False, CustomTag=
```

Comment:
Note, this order is sorted by hash code. I tried very hard to update this test to avoid this dependency on hash code, but gave up due to the number of indirections and hash tables involved. We should move to the same test ProjectTree in TestServices that the rest of our tests use.